VMBackup/main/handle.py

```diff
@@ -334,10 +334,15 @@ def daemon():
             utcNow = datetime.datetime.utcnow()
             backup_logger.log('command start time is ' + str(commandStartTime) + " and utcNow is " + str(utcNow), True)
             timespan = utcNow - commandStartTime
-            MAX_TIMESPAN = 150 * 60 # in seconds
+            MAX_TIMESPAN = 140 * 60 # in seconds
             # handle the machine identity for the restoration scenario.
             total_span_in_seconds = timedelta_total_seconds(timespan)
             backup_logger.log('timespan is ' + str(timespan) + ' ' + str(total_span_in_seconds))
+            
+            if total_span_in_seconds > MAX_TIMESPAN :
```

Comment:
This can be moved to enable as well. i checked that in enable only config files are read and logged and the main checks happen in daemon. 
For eg : if(bool(public_settings) == False and not protected_settings):
            error_msg = "unable to load certificate"

Also, keeping it here would keep it consistent with Windows extension as well, but we can discuss more offline regarding the better place to keep this check.